g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/TranslationWithPatternAction.java

```diff
@@ -56,6 +64,58 @@ public APIResponseDTO getTransPattern(TranslationWithPatternDTO data) throws Exc
 	      }
 	   }
 
+	   /**
+        * According to  object get the pattern and translation or only get the
+        * pattern when use GET method
+        * 
+        */
+	    public APIResponseDTO getTransPattern(int combine, String productName, String version,
+	            String components, String language, String scope, String region, String pseudo,
+	            HttpServletRequest req) throws Exception{
+	        // TODO Auto-generated method stub
+	        TranslationWithPatternDTO data = new TranslationWithPatternDTO();
```

Comment:
It seems the format is messy. Could you format the code?

Reply:
done